cybsi/api/observable/entities_api.py

```diff
@@ -550,14 +556,18 @@ def add_labels(self, entity_uuid: uuid.UUID, labels: Iterable[str]) -> None:
     def delete_labels(self, entity_uuid: uuid.UUID, labels: Iterable[str]) -> None:
         """Delete entity labels.
 
+         .. deprecated:: 2.14
```

Comment:
Кажется лишний отступ, здесь и ниже
При сборке документации всё хорошо отображается?

```suggestion
        .. deprecated:: 2.14
```

Reply:
Здесь после сборки доки, получается отдельный отступ получается

<img width="910" alt="image" src="https://github.com/user-attachments/assets/6c5cb59e-5e83-4af6-84d3-754ad82bd784">

А хотелось бы без отступа, как в методе добавления меток

<img width="923" alt="image" src="https://github.com/user-attachments/assets/e38d3cb6-2770-4b1c-bfed-2cbfee385feb">